File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-2.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 0
        values = set()

        i = 0;

        while i <len(nums):
            print(i)
            if nums[i] in values:
                nums.remove(nums[i])
                i -= 1
            else:
                values.add(nums[i])
                k+=1
            i +=1
        print(values)
        return k;

File: Data Structures & Algorithms/duplicate-integer/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        for(let i = 0; i < nums.length; i++){
            let curr = nums[i];
            for(let j = i + 1; j <  nums.length; j++){
                if(nums[j] ===curr){
                    return true;
                }
            }
        }


        return false;
    }
}
